students/morgan/mailprog/mailprog/mailprog/dataModels.py

```diff
@@ -0,0 +1,181 @@
+import csv
+import copy
+
+
+
+
+
+
+
+
+
+class Donor(object):
+
+    def __init__(self, name , donations= None):
+        self.name = name
+        self.donations = donations
+        self.i = 0
+
+    def add_donation(self, money):
+        self.donations.append(money)
+
+    def __iter__(self):
```

Comment:
if you use a generator here (very slick!) then you don't need the `__next__` method. or `self.i`.